src/Mappers/Root/LastTopRootTypeMapper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TheCodingMachine\GraphQLite\Mappers\Root;
+
+use GraphQL\Type\Definition\InputType;
+use GraphQL\Type\Definition\NamedType;
+use GraphQL\Type\Definition\OutputType;
+use GraphQL\Type\Definition\Type as GraphQLType;
+use phpDocumentor\Reflection\DocBlock;
+use phpDocumentor\Reflection\Type;
+use ReflectionMethod;
+use ReflectionProperty;
+
+/**
+ * The last "top" root type mapper that always calls the dynamicly set "next" mapper.
+ */
+class LastTopRootTypeMapper implements RootTypeMapperInterface
```

Comment:
Does it make sense to call it the last "top" "root" type mapper?  Or does LastTypeMapper or LastRootTypeMapper make more sense?  The name does help with understanding the dynamic nature.  However, it seems like we might be able to add further clarity in the naming?  Is it really the "top" if it's the "last" and dynamic?

Reply:
Agreed. What about "LastMutableTypeMapper" or "LastDelegatingTypeMapper"? On the "Root" suffix, all other root mappers don't have for some reason so I think I should remove it.

Reply:
`LastDelegatingTypeMapper` makes more sense to me.